### DISCUSSION TITLE
Is it possible to synchronize only OneDrive Business Shared Folders?

### DISCUSSION THREAD

**Discussion by rrodrigueznt** [abraunegg/onedrive#2282] [Q&A] [answered]
Hi! I'm running ```OneDrive v2.4.23``` on a ```CentOS Stream``` 8 server.

Can you synchronize only OneDrive Business Shared Folders? I'm reading [How to configure OneDrive Business Shared Folder Sync](https://github.com/abraunegg/onedrive/blob/master/docs/BusinessSharedFolders.md#how-to-configure-onedrive-business-shared-folder-sync) and concluding that ```--sync-shared-folders``` at running time has the same effect as the configuration option ```sync_business_shared_folders = "true"``` and that ```skip_dir``` affects both my folders and folders shared with me.

Please, am I right? Any alternative? Thanks!

**Comment by abraunegg** [comment, marked as answer]
@rrodrigueznt 
> Can you synchronize only OneDrive Business Shared Folders?

By current design, OneDrive Business Shared Folders get synced into your configured 'sync_dir', thus, 'skip_dir' will affect data from your Business OneDrive root and all the Shared folders.

To synchronize only OneDrive Business Shared Folders, you will need to run 2 x instances:
* Instance 1 syncs your Business OneDrive root
* Instance 2 syncs your OneDrive Business Shared Folders to a 'different sync_dir' to instance 1, and specifically exclude Business OneDrive root by utilising 'sync_list' to only sync your Business Shared Folders.

Test with `--dry-run` to validate the above.

**Reply by rrodrigueznt** [reply]
@abraunegg 
Thanks for your quick answer.

> * Instance 1 syncs your Business OneDrive root

I guess an instance without skipping folders or files and with ```sync_list = false``` is what will sync my Business OneDrive root. I'm happily using such a configuration in several servers. 

> * Instance 2 syncs your OneDrive Business Shared Folders to a 'different sync_dir' to instance 1, and specifically exclude Business OneDrive root by utilising 'sync_list' to only sync your Business Shared Folders.

Here I'm lost: I'm not able to figure out other way of going than including in ```sync_list``` the same contents as in ```business_shared_folders```

For instance, if my ```business_shared_folders``` reads:

```
22C10_2022-PG047.00_SABORIDO SILVA, JUAN JOSE
22C10_2022-PG047.01_SABORIDO SILVA, JUAN JOSE
22C10_2022-PG062_CAZON BOADO, LORENZO
```

Must my ```sync_list``` read the same? Thanks!

**Reply by abraunegg** [reply]
Correct - your sync_list file will need to match your business folder entries.

I have not tried this - but you *may* be able to symbolically link sync_list to business_shared_folders .... meaning one change, one location to edit
